ocaml/tests/test_vdi_cbt.ml

```diff
@@ -379,11 +379,16 @@ let test_data_destroy =
       let data_destroy ~timeout =
         (* We need a 1-second tolerance here, otherwise the test will fail *)
         let timebox_timeout = timeout +. 1.0 in
-        Helpers.timebox
-          ~timeout:timebox_timeout
-          ~otherwise:(fun () -> Alcotest.fail (Printf.sprintf "data_destroy did not return in %f seconds" timebox_timeout))
-          (fun () -> Xapi_vdi._data_destroy ~__context ~self:vDI ~timeout)
-       in
+        let success = ref false in
+        let raisedexn = ref None in
+        ignore (bg (fun () -> (try Xapi_vdi._data_destroy ~__context ~self:vDI ~timeout with e -> raisedexn := Some e); success := true));
```

Comment:
`success` is a bit ambiguous here... it seems that it means "not timed out", but an exception could still have been raised, so the function-under-test did not succeed.

Reply:
Good point, will fix this.